src/Data/IdGenerator/Masa.Contrib.Data.IdGenerator.SequentialGuid/SequentialGuidGenerator.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Data.IdGenerator.SequentialGuid;
+
+/// <summary>
+/// This code is taken from jhtodd/SequentialGuid https://github.com/jhtodd/SequentialGuid/blob/master/SequentialGuid/Classes/SequentialGuid.cs
```

Comment:
If there is no license, the copyright is available by default

Reply:
processed